### DISCUSSION TITLE
In-game interface (HUD) appearing and disappearing randomly, even when playing with "Never" or "Hide during gameplay"

### DISCUSSION THREAD

**Discussion by TiDeane** [ppy/osu#27060] [Q&A] [answered]
Hiding the HUD during gameplay doesn't seem to work, as it flashes even when playing with "Never" or "Hide during gameplay"

Video example:
https://www.youtube.com/watch?v=xRQLZYIRryU

Testing each setting: "Hide during gameplay" starts at 0:04, "Always" starts at 0:47, "Never" starts at 0:51, and back to "Hide during gameplay" at 1:08

Any idea what causes this?

**Comment by bdach** [comment]
Are you pressing ctrl?

**Reply by TiDeane** [reply]
No, just Z and X for the gameplay. I do hover my pinky finger over ctrl while playing, but I don't press it. Could it be related to that?

**Reply by bdach** [reply, marked as answer]
Well if you _were_ to press ctrl accidentally that could cause the hud to display because when you hold ctrl the hud is shown momentarily regardless of the setting.

**Reply by TiDeane** [reply]
That was it, thank you! Should I delete the original post or should I keep it?

**Reply by bdach** [reply]
It can stay for others to see.
